tests/Moq.Tests/MockRepositoryFixture.cs

```diff
@@ -52,6 +52,61 @@ public void ShouldVerifyAll()
 			}
 		}
 
+		[Fact]
+		public void ShouldVerifyNoOtherCalls()
+		{
+			var repository = new MockRepository(MockBehavior.Default);
+			repository.Create<IFoo>();
+
+			repository.VerifyNoOtherCalls();
+		}
+
+		[Fact]
+		public void ShouldVerifyNoOtherCalls_WhenUnmatched()
+		{
+			var repository = new MockRepository(MockBehavior.Default);
+			var mock = repository.Create<IFoo>();
+
+			mock.Object.Do();
+
+			var mex = Assert.Throws<MockException>(() => repository.VerifyNoOtherCalls());
+			Assert.Equal(MockExceptionReason.UnverifiedInvocations, mex.Reason);
+		}
+
+		[Fact]
+		public void ShouldVerifyNoOtherCalls_WhenVerified()
+		{
+			var repository = new MockRepository(MockBehavior.Default);
+			var mock = repository.Create<IFoo>();
+
+			mock.Object.Do();
+
+			mock.Verify(foo => foo.Do(), Times.Once);
+			repository.VerifyNoOtherCalls();
+		}
+
+		[Fact]
+		public void VerifyNoOtherCalls_should_correctly_aggregate_unmatched_calls_from_more_than_one_mock()
+		{
+			var repository = new MockRepository(MockBehavior.Default);
+			var fooMock = repository.Create<IFoo>();
+			var barMock = repository.Create<IBar>();
+
+			fooMock.Object.Do();
+			barMock.Object.Redo();
+
+			var ex = Record.Exception(() => repository.VerifyNoOtherCalls());
+
+			var expectedErrorMessage = $"The following invocations on mock \'{fooMock}\' were not verified:\r\n" +
+									   "IFoo.Do()\r\n" +
+									   "\r\n" +
+									   $"The following invocations on mock \'{barMock}\' were not verified:\r\n" +
+									   "IBar.Redo()\r\n";
```

Comment:
This and one other test currently fail on AppVeyor. I suspect this is due to your use of `"\r\n"` (CRLF) which isn't used on all system. Please try replacing it with `Environment.NewLine`.

Reply:
Facepalm!

Reply:
Seems I wasn't completely right about this, obviously `Resources.resx` embeds string resources with different line endings than what `Environment.NewLine` reports. 😒 Since I've sort-of led you down the wrong rabbit hole, I'll take responsibility and fix this (and other tests that compare multi-line strings).